projects/igniteui-angular/src/lib/directives/mask/mask.directive.ts

```diff
@@ -56,21 +55,6 @@ export class IgxMaskDirective implements OnInit, AfterViewChecked, ControlValueA
     @Input()
     public includeLiterals: boolean;
 
-    /**
-     * Specifies a placeholder.
-     * ```html
-     * <input placeholder = "enter text...">
-     * ```
-     */
-    @DeprecateProperty('"placeholder" is deprecated, use native placeholder instead.')
-    public set placeholder(val: string) {
-        this.renderer.setAttribute(this.nativeElement, 'placeholder', val);
-    }
-
-    public get placeholder(): string {
```

Comment:
I'm not sure, but I think we should add a **breaking change** for this in the CHANGELOG